refinery/tool_manager/tests.py

```diff
@@ -1046,6 +1078,69 @@ def test_bad_POST_transaction_rollback(self):
         self.assertIn("LIST/PAIR structure is not balanced",
                       self.post_response.content)
 
+    def test_bad_extra_directories_path(self):
+        Tool.objects.all().delete()
+        with open("{}/visualizations/"
+                  "LIST_visualization_bad_extra_directories_path.json"
+                  .format(TEST_DATA_PATH)) as f:
+            visualization_annotation = json.loads(f.read())
+            create_tool_definition(visualization_annotation)
+
+        td = ToolDefinition.objects.get(
+            name=visualization_annotation["name"]
+        )
+
+        tool_launch_configuration = {
+            "tool_definition_uuid": td.uuid,
+            "file_relationships": str(["www.example.com"])
```

Comment:
Not sure how it's used, but this seems like a url, not a file? and is a serialized python array the best choice? Maybe represent each separately, but if this is best, perhaps the name should be more explicit?